.github/CONTRIBUTING.md

```diff
@@ -34,151 +30,191 @@ ensure that the community is free to use your contributions.
-build in parallel, using 8 processes.
+    qiskit-aer$ python ./setup.py bdist_wheel -- -j8
 
-*N.B. on MacOS:*, you may need to turn off static linking and specify your platform name, e.g.:
+This is setting the **CMake** variable `STATIC_LINKING` to value `True` so
+**CMake** will try to create an statically linked **cython** library, and is
+passing `-j8` flag to the underlaying build system, which in this case is
+Makefile, telling it that we want to build in parallel, using 8 processes.
 
-```
-qiskit-aer$ python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- -DSTATIC_LINKING=False -- -j8
-```
+> You may need to turn off static linking and specify your platform name, e.g.:
 
-After this command is executed successfully, we will have a wheel package into the ``dist/`` directory, so next step is installing it:
+>   qiskit-aer$ python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- >   -DSTATIC_LINKING=False -- -j8
 
+After this command is executed successfully, we will have a wheel package into
+the `dist/` directory, so next step is installing it:
 
-```
-qiskit-aer/$ cd dist
-qiskit-aer/dist$ pip install qiskit_aer-<...>.whl
-```
+    qiskit-aer/$ cd dist
+    qiskit-aer/dist$ pip install qiskit_aer-<...>.whl
 
-**Standalone executable**
+**Standalone Executable**
 
-If we want to build an standalone executable, we have to use CMake directly.
-The preferred way CMake is meant to be used, is by setting up an "out of source" build.
-So in order to build our standalone executable, we have to follow these steps:
+If we want to build a standalone executable, we have to use **CMake** directly.
+The preferred way **CMake** is meant to be used, is by setting up an "out of
+source" build. So in order to build our standalone executable, we have to follow
+these steps:
 
-All platforms
+    qiskit-aer$ mkdir out
+    qiskit-aer$ cd out
+    qiskit-aer/out$ cmake ..
+    qiskit-aer/out$ cmake --build . --config Release -- -j4
 
+Once built, you will have your standalone executable into the `Release/` or
+`Debug/` directory (depending on the type of building chosen with the `--config`
+option):
 
-```
-qiskit-aer$ mkdir out
-qiskit-aer$ cd out
-qiskit-aer/out$ cmake ..
-qiskit-aer/out$ cmake --build . --config Release -- -j4
-```
+    qiskit-aer/out$ cd Release
+    qiskit-aer/outRelease/$ ls
+    aer_simulator_cpp
 
-Once built, you will have your standalone executable into the ``Release`` or ``Debug``
-directory (depending on the type of building choosen with the ``--config`` option):
 
 
-```
-qiskit-aer/out$ cd Release
-qiskit-aer/out/Release$ ls
-aer_simulator_cpp
-```
+### Windows
+
+On Windows, you must have **Anaconda3** installed. We recommend also installing
+**Visual Studio 2017** (Community Edition). **Anaconda3** is required when
+searching for an **OpenBLAS** implementation. If **CMake** can't find a suitable
+implementation installed, it will take the **BLAS** library from the
+**Anaconda3** environment.
```

Comment:
Ok, this is my fault. The instructions are very poor, I need to improve Windows instructions.